queue.tf

```diff
@@ -26,6 +26,14 @@ module "notification-queue" {
   lock_duration       = "PT5M"
 }
 
+module "processed-envelope-queue" {
+  source              = "git@github.com:hmcts/terraform-module-servicebus-queue.git"
+  name                = "processed-envelopes"
```

Comment:
it would be nice for the name to match the module name (plural vs singular 'envelope')

Reply:
I totally agree. Chose to follow existing convention, though.

Reply:
On a second thought, I'll change the convention.